scripts/skills/perks/perk_rf_decisive.nut

```diff
@@ -0,0 +1,160 @@
+this.perk_rf_decisive <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		FatigueMult = 0.85,
+		ResolveModifier = 15,
+		DamageMult = 1.15,
+		AttractionMult = 1.1,
+
+		// Const
+		Stacks = 0,
```

Comment:
`Stacks` really isn't "Const", it is variable. I can't think of an alternative name though. I also think that commenting these fields like this is overkill and doesn't add any value. By some logic `MaxStacks` is also "config" just like `FatigueMult`. And `FatigueMult` is a "const" as much as it is "config". I understand the desire to separate the mults/modifiers from the "stacks" type fields, in which case a simple empty line between the two would be sufficient e.g.
```squirrel
FatigueMult = 0.85,
ResolveModifier = 15,
DamageMult = 1.15,
AttractionMult = 1.1,

Stacks = 0,
MaxStacks = 3
```

Reply:
I improved the formatting and removed the `// Const` category. 
In general I'm happy with the use of comments to separate the member variables.